### ISSUE TITLE
scrollIntoView options

### ISSUE THREAD

**Issue by RyanMulready** [pulsardev/vue-tour#34]
It would be useful if we were able to pass in the options we want to when the `scrollIntoView` method is triggered.   It could be just props on the component `scrollIntoViewOptions` or something.

**Comment by Outpox** [comment]
Yep that's a good idea, I believe it may help. I'll see what I can do ;-)
